src/main.js

```diff
@@ -60,3 +70,18 @@ function viewHomePage() {
   randomCoverButton.classList.remove('hidden');
   saveCoverButton.classList.remove('hidden');
 }
+
+function displayFormSubmissionCover(event) {
+  event.preventDefault();
+  updatedCoverImage.src = inputCoverImage.value;
+  updatedTitle.innerText = inputTitle.value;
+  updatedDescriptorOne.innerText = inputDesc1.value;
+  updatedDescriptorTwo.innerText = inputDesc2.value;
+  covers.push(inputCoverImage.value);
+  titles.push(inputTitle.value);
+  descriptors.push(inputDesc1.value, inputDesc2.value);
+  currentCover = new Cover(inputCoverImage.value, inputTitle.value, inputDesc1.value, inputDesc2.value);
+  formViewPage.classList.add('hidden');
+  mainCoverPage.classList.remove('hidden');
+  saveCoverButton.classList.remove('hidden');
```

Comment:
I see these 3 elements have a class being added and removed in a lot of places in the code. I think this would be a good opportunity to consider instead using `.toggle('hidden')` and pulling them into their own function instead so that you can keep your code DRY.

Reply:
That sounds great! We will definitely go back and do this!